go.mod

```diff
@@ -418,7 +418,7 @@ replace (
 	k8s.io/component-base => k8s.io/kubernetes/staging/src/k8s.io/component-base v0.0.0-20220524121711-3ddd0f45aa91
 	k8s.io/component-helpers => k8s.io/kubernetes/staging/src/k8s.io/component-helpers v0.0.0-20220524121711-3ddd0f45aa91
 	k8s.io/controller-manager => k8s.io/kubernetes/staging/src/k8s.io/controller-manager v0.0.0-20220524121711-3ddd0f45aa91
-	k8s.io/cri-api => k8s.io/kubernetes/staging/src/k8s.io/cri-api v0.0.0-20210928072042-90051602459c
+	k8s.io/cri-api => k8s.io/kubernetes/staging/src/k8s.io/cri-api v0.0.0-20220626121858-f4abde9e57cc
```

Comment:
We create 1.25.0-alpha.2 in k/k today, let's update every replaced component to this tag/commit to avoid vendoring incompatible packages.

Reply:
thanks.  I've pushed a new version now